docs/concepts/worklist.md

```diff
@@ -0,0 +1,140 @@
+
+The `Accept` and `Content-Type` headers are required in the request, and must both have the value `application/json`.
+
+There are a number of DICOM elements which are required to be present in every Workitem dataset. The full list is in
+[this table](https://dicom.nema.org/medical/dicom/current/output/html/part04.html#table_CC.2.5-3).
+
+Notes on dataset attributes:
+- **SOP Instance UID:** Although the reference table above says that SOP Instance UID should not be present, this guidance is specific to the DIMSE protocol and is
+handled diferently in DICOMWeb&trade;. SOP Instance UID **should be present** in the dataset if not in the URI.
+- **Scheduled Procedure Step State:** this attribute must be present, but must have no value.
+
+### Store Response Status Codes
+
+| Code                         | Description |
+| :--------------------------- | :---------- |
+| 201 (Created)                | The target Workitem was successfully created. |
+| 400 (Bad Request)            | There was a problem with the request. For example, the request payload did not satisfy the requirements above. |
+| 401 (Unauthorized)           | The client is not authenticated. |
+| 409 (Conflict)               | The Workitem already exists. |
+| 415 (Unsupported Media Type) | The provided `Content-Type` is not supported. |
+| 503 (Service Unavailable)    | The service is unavailable or busy. Please try again later. |
+
+### Store Response Payload
+
+A success response will have no payload. The `Location` and `Content-Location` response headers will contain
+a URI reference to the created Workitem.
+
+A failure response payload will contain a message describing the failures.
+
+## Search
+
+This transaction enables you to search for Workitems by attributes.
+
+| Method | Path                                            | Description                       |
+| :----- | :---------------------------------------------- | :-------------------------------- |
+| GET    | ../workitems?                                   | Search for Workitems              |
+
+The following `Accept` header(s) are supported for searching:
+
+- `application/json`
+
+### Supported Search Parameters
+
+The following parameters for each query are supported:
+
+| Key              | Support Value(s)              | Allowed Count | Description |
+| :--------------- | :---------------------------- | :------------ | :---------- |
+| `{attributeID}=` | {value}                       | 0...N         | Search for attribute/ value matching in query. |
+| `includefield=`  | `{attributeID}`<br/>`all`   | 0...N         | The additional attributes to return in the response. Both public and private tags are supported.<br/>When `all` is provided, please see [Search Response](###Search-Response) for more information about which attributes will be returned for each query type.<br/>If a mixture of {attributeID} and 'all' is provided, the server will default to using 'all'. |
```

Comment:
For include field, should we mention that we only top level fields and not nested fields